plugins/page_params/Pageparams_view.php

```diff
@@ -218,6 +218,43 @@ function Pageparams_linkList($default, $disabled)
     return $o;
 }
 
+/**
+ * Returns a header status selectbox.
+ *
+ * @param array $page Page data of the current page.
+ *
+ * @return string HTML
+ *
+ * @since 1.8
+ */
+function Pageparams_headerStatusSelectbox($page_header_status)
+{
+
+    if (isset($page_header_status) && trim($page_header_status) !== '') {
+        $header_status = $page_header_status;
+        $selected = '';
+    } else {
+        $header_status = '';
+        $selected = ' selected="selected"';
+    }
+    $o = "\n" . '<select name="header_status">';
+    $o .= "\n\t" . '<option value=""' . $selected . '></option>';
+    $options = array('403', '404', '410');
+    foreach ($options as $header_status_tmp) {
+        $selected = ($header_status_tmp == $header_status) ? ' selected="selected"' : '';
+        $o .= "\n\t"
+            . '<option value="'
+            . $header_status_tmp
+            . '"'
+            . $selected
+            . '">' 
```

Comment:
Da ist ein überflüssiges Leerzeichen am Zeilenende, das der CodeSniffer bemängelt. Vielleicht kann das so behoben werden:
```suggestion
            . '">'
```

Reply:
Es kann so einfach sein.
Danke